addon/serializer.js

```diff
@@ -8,7 +8,7 @@ export default JSONAPISerializer.extend({
     return underscore(attr);
   },
 
-  keyForRelationship: function(rawKey, kind) {
+  keyForRelationship: function(rawKey) {
```

Comment:
I think I'd rather turn off this jshint check then remove these unused arguments. Mainly because if we ever need to change this code is easier to see the argument is already there then have to look it up from the api docs.